frontend/src/pages/ProfileDeletion.tsx

```diff
@@ -15,7 +31,14 @@ export function ProfileDeletion() {
         profile? We're sad to see you go! Please remember, this action is
         permanent and you'll lose all your data."
       </p>
-      <button type="button" style={{ backgroundColor: "red", margin: "10px" }}>
+      <span style={{ fontWeight: "bold", color: "red", display: "block" }}>
+        {errMessage}
+      </span>
```

Comment:
Technically the `<span>`-s should be omitted, when no content is present. On the other hand it is not very react-ish. You should rather use conditional rendering based on the variables that you are referring to.